### DISCUSSION TITLE
JCenter is closing: find a solution for screengrab

### DISCUSSION THREAD

**Discussion by guillaume-tgl** [fastlane/fastlane#18166] [Ideas] [answered]
JCenter is being closed in May: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter

Screengrab won't be available on this repository after that date. It would be great to publish it somewhere else, like on MavenCentral. [This issue](https://github.com/fastlane/fastlane/issues/12651) was opened but is now locked, it should probably reopened.

**Comment by joshdholtz** [comment, marked as answer]
Hello!

I reopened the issue 😊 I will be getting things migrated over to maven. Good news is the jcenter repository will be served until 2/1/22 but we do want to make the change as soon as possible 💪 

> To clarify, the JCenter repository will keep serving packages for 12 months until February 1st 2022. Only the JCenter REST API and UI will be sunsetted on May 1st 2021.

Thank you for bringing this up! ❤

**Comment by penn5** [comment]
I just recently made a PR fixing bugs in the screengrab code, and spent almost 3 hours trying to make JCenter work... :(
